### DISCUSSION TITLE
How to revoke a refresh token?

### DISCUSSION THREAD

**Discussion by mongeon** [aws/aws-sdk-net-extensions-cognito#99] [Q&A] [answered]
Is there an easy way to revoke a refresh token?

**Comment by ashishdhingra** [comment, marked as answer]
@mongeon Please refer [Revoking tokens](https://docs.aws.amazon.com/cognito/latest/developerguide/token-revocation.html). After enabling token revocation in user pool client (this could be done in AWS Console for a user pool, under `General Settings` > `App clients` > `<<app client>>` > `Advanced token settings` > `Enable token revocation`), you may use `AWSSDK.CognitoIdentityProvider` package, to execute `AmazonCognitoIdentityProviderClient.RevokeTokenAsync()` call.

Thanks,
Ashish

[a comment by an automated account is not shown]
